### PR TITLE
Bug fix/7904 getting exception while running flow

### PR DESCRIPTION
**Pull request by sudarshan0212**

### CODE REVIEW

File: Ginger/GingerCoreNETUnitTest/RunTestslib/ActReturnValueTest.cs [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+            act.AddNewReturnParams = true;
+            act.SupportSimulation = true;
+            mGingerRunner = new GingerRunner();
+            mGingerRunner.RunInSimulationMode = true;
+            BF.Activities[0].Acts.Add(act);
+            mGingerRunner.BusinessFlows.Add(BF);
+        }
+      
+        [TestMethod]
+        public void Contains()
```
Comment (JinendraGhodke) [inline Ginger/GingerCoreNETUnitTest/RunTestslib/ActReturnValueTest.cs]: Change the test names to something like OutputValueContainsTest

File: Ginger/GingerCoreNETUnitTest/RunTestslib/ActReturnValueTest.cs [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+       
+        [TestMethod]
+        public void Equals()
+        {
+            ARC.SimulatedActual = "Test";
+            ARC.Expected = "Test";
+            ARC.Operator = Amdocs.Ginger.Common.Expressions.eOperator.Equals;
+            act.ActReturnValues.Add(ARC);
+            mGingerRunner.RunRunner();
+            Assert.AreEqual(act.Status, Amdocs.Ginger.CoreNET.Execution.eRunStatus.Passed);
```
Comment (JinendraGhodke) [inline Ginger/GingerCoreNETUnitTest/RunTestslib/ActReturnValueTest.cs]: Add validation for ARC status as well

Comment (sudarshan0212) [reply Ginger/GingerCoreNETUnitTest/RunTestslib/ActReturnValueTest.cs]: Added ARC status check

File: Ginger/GingerCoreNETUnitTest/RunTestslib/ActReturnValueTest.cs [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GingerCoreNETUnitTest.RunTestslib
+{
+    [TestClass]
+    public class ActReturnValueTest
+    {
+        static ActDummy act;
```
Comment (JinendraGhodke) [inline Ginger/GingerCoreNETUnitTest/RunTestslib/ActReturnValueTest.cs]: Please create a separate act for each unit test.
Using same object may create issue when running tests in parallel

**Pull request merged**